tests/unit/transform/date-test.js

```diff
@@ -39,28 +38,22 @@ test('#deserialize', function(assert) {
   assert.equal(transform.deserialize(undefined), null);
 });
 
-test('#deserialize with different offset formats', function(assert) {
+testInDebug('#deserialize with different offset formats', function(assert) {
```

Comment:
I agree, but I'm not sure (based on how that stripping happens) how to set this up at a glance. I will think.